menu.html

```diff
@@ -1,6 +1,6 @@
 
-<button id="homeButton" class="navigationButtons" onclick="window.location.href = 'page/home.html'">Home</button>
-<button id="teamButton" class="navigationButtons" onclick="window.location.href = 'page/zimmer.html'">Zimmer</button>
-<button id="hireUsButton" class="navigationButtons" onclick="window.location.href = 'page/buchen.html'">Buchen</button>
-<button id="imprintButton" class="navigationButtons" onclick="window.location.href = 'page/ueberUns.html'">Über uns</button>
-<button id="kontaktButton" class="navigationButtons" onclick="window.location.href = 'page/kontakt.html'">Kontakt</button>
\ No newline at end of file
+<button id="homeButton" class="navigationButtons" onclick="window.location.href = '?page=home'">Home</button>
+<button id="teamButton" class="navigationButtons" onclick="window.location.href = '?page=zimmer'">Zimmer</button>
+<button id="hireUsButton" class="navigationButtons" onclick="window.location.href = '?page=buchen'">Buchen</button>
+<button id="imprintButton" class="navigationButtons" onclick="window.location.href = '?page=ueberUns'">Über uns</button>
+<button id="kontaktButton" class="navigationButtons" onclick="window.location.href = '?page=kontakt'">Kontakt</button>
```

Comment:
See here how I changed the URLs. This needs to be done in the whole application.